File: Makro.IMS/Makro.IMS.Infra.Data/Models/ReportSlottimeBooking.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection.Metadata.Ecma335;
using System.Security.Permissions;
using System.Text;
using System.Threading.Tasks;

namespace Makro.IMS.Infra.Data.Models
{
    public class ReportSlottimeBooking
    {
        public DateTime BookingDate { get; set; }
        public string? MerchType { get; set; }
        public string? DataType { get; set; }
        public DateTime SlotTime { get; set; }
        public decimal? Quantity { get; set; }
        
    }

}

File: Makro.IMS/Makro.IMS.Web/src/app/@core/models/report/report-slottimebooking.ts
interface ReportSlottimeBooking{
    bookingDate: Date;
    merchType: string;
    dataType: string;
    slotTime: Date;
    quantity: number;
}

export {ReportSlottimeBooking};

interface ReportSlottimeBookingAllWhse{
    bookingDate: Date;
    merchType: string;
    dataType: string;
    slotTime: Date;
    quantity: number;
    warehouseCode: string;
}

export {ReportSlottimeBookingAllWhse};